src/stylesheets/demo.scss

```diff
@@ -0,0 +1,2 @@
+@import "../../node_modules/bootstrap/dist/css/bootstrap.min.css";
```

Comment:
@adriana-alter just want to point out that the components still depend on the base styling from bootstrap. For the demo, it's needed.

Reply:
My goal for this iteration is just to get away from react-bootstrap, not regular bootstrap.  But in the next pass...